styles.css

```diff
@@ -75,14 +76,34 @@ body { font-family: 'Roboto', Arial, Helvetica, Sans-serif; }
 
 .nav__link:last-child { margin-right: 0; }
 
-.main-section { padding-top: 245px; }
+.main-section {
+  padding: 0 67% 100px 0;
+  background: top right no-repeat url(img/frame.svg);
+}
+
+.main-section__title {
+  padding: 0 0 40px 0;
+  font: bold 40px/55px 'pt_serif', arial, helvetica, sans-serif;
+}
 
-.main-section__title { font-family: 'pt_serif', arial, helvetica, sans-serif; }
+.main-section__title_highlight { background: #ecf3ff; }
 
-.main-section__text { padding: 40px 0; }
+.main-section__text {
+  padding: 0 0 41px 0;
+  font: 18px/21px 'Roboto', arial, helvetica, sans-serif;
+}
 
 .main-section__btn {
-  padding: 29px 140px;
+  padding: 14px 30px;
   background-color: #0056ec;
   color: white;
+  font: 14px/16px 'Roboto', arial, helvetica, sans-serif;
+  display: inline-block;
+  text-decoration: none;
+  border-radius: 4px;
+  transition: opacity .5s;
 }
+
+.main-section__btn:hover {
+  opacity: .5;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/html_2.0/pullRequest?prid=6361385)